0004/README.md

```diff
@@ -0,0 +1,416 @@
+and follow its terms.  
+
+The SLSA community provides a security questionnaire for build platforms. This
+questionnaire is the minimum standard for evidence of SLSA conformance.
+Verifiers and auditors are encouraged to seek evidence beyond the questionnaire
+when making their trust determinations. For a draft of the questionnaire, see
+Appendix A.
+
+### Build platform participation
+
+There are two tiers for a build platform to take part in the program:
+self-attestation and third-party audit. Self-attestation is a weaker proof of
+conformance than a third-party audit, and is most useful while bootstrapping the
+conformance program before auditors are available.
+
+#### Tier one: Self-attestation
+
+The self-attestation tier requires that a build platform:
+
+-  Publish answers to the questionnaire on a domain registered to their
+    organization. They may additionally publish further proof of SLSA
+    conformance (e.g. architecture documentation). This information must be
+    updated at least once every twelve months and any time there is an
+    architectural change to the build platform.
+-  Publish any materials needed to verify provenance signed by the build
+    system on a domain registered to their organization.
+-  Publish vulnerability disclosures for all security incidents that
+    potentially impact the integrity of artifacts or provenance generated by
+    the build system.
+-  Sign and follow the SLSA Conformance Program terms.
+
+The OpenSSF operates this tier through a community-maintained repository of
+build platforms. The maintainers of this repository will be drawn from members
+of the SLSA community who are spread across the industry.  
+In order to participate in this tier, build platforms should 
+
+1. Learn about the SLSA Specification and its requirements.
+1. Create a PR for the repository containing
+    1. Name of the build platform
+    1. Build ID
+    1. Desired SLSA levels
+    1. List of URIs that resolve to proof of SLSA conformance
+    1. List of URIs that resolve to materials for verifying provenance
+    1. List of URIs that resolve to the build platform's public
+        vulnerability disclosures
+
+1. Complete the terms and conditions form.
+1. Send the PR for review.
+
+The community will review the PR and merge it when a quorum of maintainers
+approve it. When reviewing the PR, the community will assume that the
+information in the proof of conformance is truthful, and they will approve it if
+the described build platform meets the requirements for the desired SLSA levels.
+
+Once the PR is merged and the terms are signed, the OpenSSF grants the build
+platform a limited license to use the SLSA logo on its website and other
+promotional materials. Should the build platform fail to adhere to the program's
+terms or meet its requirements, the OpenSSF will revoke that license and, if
+necessary, enforce its trademarks. 
+
+#### Tier two: Third-party audit
```

Comment:
I propose we actually create a new in-toto [predicate](https://github.com/in-toto/attestation/tree/main/spec/predicates) which can be used by third-party auditors to sign new in-toto attestations to the effect that they vouch for the given level of SLSA compliance for a build platform. I see that a build platform attestation has been proposed and rejected below, the latter of which I believe is too early to do so.

Reply:
What would go in this attestation, and who is verifying the signature? Is the idea to verify the attestation as part of provenance verification, or would you do it once as part of adding a trust root to a verifier tool?

Reply:
Good Qs. I imagine it's something you download, cache, and verify every now and then (although it does need to come with an expiration for obvious reasons).